src/views/Swap/Swap.vue

```diff
@@ -651,6 +668,14 @@ export default {
         [this.assetChain]: this.asset,
         [this.toAssetChain]: this.toAsset
       }
+    },
+    checkHighFee () {
```

Comment:
since it will become a property, it should not be named with an action.
more like "isFeeTooHigh"
same with other one "isSwapNegative"